dist/OBS-statistics.R

```diff
@@ -0,0 +1,291 @@
+# We use ggplot2, a library that makes painting graphics much easier.
+# You can check the types of graphs and examples code here: https://plot.ly/ggplot2
+
+# REQUIRE NEEDED LIBRARIES, in case that you don't have then installed, use: install.packages("ggplot2")
```

Comment:
Just wondering, do we really want to include an R script to generate graphs in this project? Since this is an open source project IMO it seems weird to include an R script which was used for a presentation on this project.